### PR TITLE
2020 Conference Updates

### PR DESCRIPTION
**Pull request by antonjb**
Removes and archives 2019 content setting up the site to not show old
information and getting ready for filling out more of 2020.

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
         since 2018 it has been run by DDD WA Inc.
       </SafeLink>
       . We are now in our fifth year, which we think is worth celebrating!{' '}
```
Comment (mzaatar) [inline pages/about.tsx]: Do we need to change this text `We are now in our fifth year`?

File: config/conference.ts [hunk 2]
```diff
@@ -25,7 +25,9 @@ const presentationSubmissionsOpenFrom = moment.parseZone('2019-04-30T08:00:00+08
 const presentationSubmissionsOpenUntil = moment.parseZone('2019-06-02T23:59:59+08:00')
 const votingOpenFrom = moment.parseZone('2019-06-08T17:00:00+08:00')
 const votingOpenUntil = moment.parseZone('2019-06-16T23:59:59+08:00')
-const agendaPublishedFrom = moment.parseZone('2019-06-23T17:00:00+08:00')
```
Comment (mzaatar) [inline config/conference.ts]: Should we change these dates as well or make them NA? not sure

File: config/conference.ts [hunk 3]
```diff
@@ -25,7 +25,9 @@ const presentationSubmissionsOpenFrom = moment.parseZone('2019-04-30T08:00:00+08
 const presentationSubmissionsOpenUntil = moment.parseZone('2019-06-02T23:59:59+08:00')
 const votingOpenFrom = moment.parseZone('2019-06-08T17:00:00+08:00')
 const votingOpenUntil = moment.parseZone('2019-06-16T23:59:59+08:00')
```
Comment (mzaatar) [inline config/conference.ts]: Should we change these dates? I Assume that we only know the conference date. What should we do about the rest of the dates? not sure

Comment (robdmoore) [reply config/conference.ts]: We aren't locking them in until March.

File: config/conference.ts [hunk 4]
```diff
@@ -25,7 +25,9 @@ const presentationSubmissionsOpenFrom = moment.parseZone('2019-04-30T08:00:00+08
...
-const agendaPublishedFrom = moment.parseZone('2019-06-23T17:00:00+08:00')
+// This is not the date for 2020 - using it to hide the agenda for now.
+// TODO: Update to actual date when known
+const agendaPublishedFrom = moment.parseZone('2020-06-23T17:00:00+08:00')
```
Comment (robdmoore) [inline config/conference.ts]: We need to put last years agenda in /agenda/2019, but assume that'll come in a different PR?

File: config/conference.ts [hunk 5]
```diff
@@ -93,8 +95,8 @@ const Conference: IConference = {
   TicketPurchasingOptions: ticketPurchasingOptions,
   HashTag: 'DDDPerth',
   SellingPoints: ['One day', 'Fully catered', 'Inclusive atmosphere', 'Interesting presentations', 'Awesome people'],
-  Handbook: 'handbook2019.pdf',
-  SessionizeUrl: 'https://sessionize.com/dddperth2019',
+  Handbook: null,
+  SessionizeUrl: null,
   SessionizeEditUrl: 'https://sessionize.com/app/speaker/',
   PreviouslySubmittedTopics:
     'Agile, building great teams, UI design, UX, software testing, virtual reality, women in tech, web accessibility, open source software, workplace culture, mental health, unconscious bias, building engaged teams, self-employment tips, mentoring, Scrum, pair programming, bots, IoT, machine learning, neural networks, quantum encryption, security, reverse engineering, blockchain, Assembly language, automated deployments, mobile development, mobile test automation, Domain Driven Design, cloud native, architecture, microservices, serverless, service meshes, stream programming and Rx, APIs, GraphQL, actors, JavaScript, SPAs, TypeScript, authentication, React, UWP, Elm, HTML, HTTP protocol, Git, Docker and pointers',
```
Comment (robdmoore) [inline config/conference.ts]: At some point we may want to update this list, I have been every year. Alternatively, we may just want to generalise it and shorten it, unsure.

File: pages/about.tsx (hunk 1 above)
Comment (robdmoore) [reply pages/about.tsx]: Yeah we can probably remove this sentence.

File: config/conference.ts (hunk 4 above)
Comment (antonjb) [reply config/conference.ts]: yeah, the change in how the agenda works has made that step a little more difficult. Will add a task to move it across.

File: pages/about.tsx (hunk 1 above)
Comment (antonjb) [reply pages/about.tsx]: Cool, have removed it.

File: config/conference.ts (hunk 3 above)
Comment (antonjb) [reply config/conference.ts]: Unfortunately the only date we have programmed to hide is the conference date. Would have been a bit of extra work to get this. Perhaps a good post conference item to improve the site.

**Pull request merged**